src/functions.js

```diff
@@ -109,13 +148,29 @@ function sales( cars ){
   //   'Ford': 20000,
   //   'Vauxhall': 15000
   // }
+  var total = {};
+  
+  cars.forEach(function(el){
+    total[el.make] = total[el.make] || 0;
+    total[el.make] += el.price;
+  });
+  return total;
 }
 
 // Harder challenges
 function secondLargest( numbers ){
   // numbers is an array of numbers
   // return the index of the second 
   // largest number in the array
+  var max = numbers.reduce(function(a, b) {return Math.max(a, b)});
+  numbers.splice(numbers.indexOf(max), 1);
+  var secondMax = numbers.reduce(function(a, b) {return Math.max(a, b)});
+
+  for(var i = 0; i < numbers.length; i++){
```

Comment:
This is a slightly inefficient way of acheiving this. `return numbers.indexOf(secondMax)` I think would work too

Reply:
Thanks Ali!